README.md

```diff
@@ -31,23 +30,27 @@ transpiled es6 classes rather than `createClass` as of v6.
 
 ## Sections covered in this README
 
-* [Installation](#installation)
-* [Usage:](#usage)
-  * [ES6 Class Component](#es6-class-component)
-  * [Functional Component with UseState Hook](#functional-component-with-usestate-hook)
-  * [CommonJS Require](#commonjs-require)
-* [Ensuring there's a click handler](#ensuring-there-is-a-click-handler)
-* [Regulate which events to listen for](#regulate-which-events-to-listen-for)
-* [Regulate whether or not to listen for outside clicks](#regulate-whether-or-not-to-listen-for-outside-clicks)
-* [Regulate whether or not to listen to scrollbar clicks](#regulate-whether-or-not-to-listen-to-scrollbar-clicks)
-* [Regulating `evt.preventDefault()` and `evt.stopPropagation()`](#regulating-evtpreventdefault-and-evtstoppropagation)
-* [Marking elements as "skip over this one" during the event loop](#marking-elements-as-skip-over-this-one-during-the-event-loop)
-* [Older React code: "What happened to the Mixin??"](#older-react-code-what-happened-to-the-mixin)
-  * [But how can I access my component? It has an API that I rely on!](#but-how-can-i-access-my-component-it-has-an-api-that-i-rely-on)
-* [Which version do I need for which version of React?](#which-version-do-i-need-for-which-version-of-react)
-  * [Support-wise, only the latest version will receive updates and bug fixes.](#support-wise-only-the-latest-version-will-receive-updates-and-bug-fixes)
-* [IE does not support classList for SVG elements!](#ie-does-not-support-classlist-for-svg-elements)
-* [I can't find what I need in the README](#i-cant-find-what-i-need-in-the-readme)
+* [:warning: Open source is free, but developer time isn't :warning:](#warning-open-source-is-free-but-developer-time-isnt-warning)
```

Comment:
Please restore the original ToC. A ToC is for indexing main content, not "every heading in the document". Anything preceding the ToC does not go in it.